src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/SubjectAccessRequestService.kt

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.goal.Goal
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.goal.service.ActionPlanService
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.Induction
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.InductionNotFoundException
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.induction.service.InductionService
+import uk.gov.justice.hmpps.kotlin.sar.HmppsPrisonSubjectAccessRequestService
+import uk.gov.justice.hmpps.kotlin.sar.HmppsSubjectAccessRequestContent
+import java.time.LocalDate
+import java.time.ZoneOffset
+
+@Service
+class SubjectAccessRequestService(
```

Comment:
I _think_ the classname needs to be suffixed with `Test` in order for it to be picked up by the test runner via the build

I could be wrong about that, but for consistency with all our other tests, it should have that class name suffix. Can we rename it `SubjectAccessRequestServiceTest` please?